osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -49,13 +49,23 @@ public partial class BeatmapCarousel : CompositeDrawable, IKeyBindingHandler<Glo
         /// </summary>
         public Action? BeatmapSetsChanged;
 
+        /// <summary>
+        /// Triggered after filter conditions have finished being applied to the model hierarchy.
+        /// </summary>
+        public Action? FilterApplied;
```

Comment:
There was no way previously to hook an event to after a filter operation had finished.